hbase-server/src/test/java/org/apache/hadoop/hbase/master/http/TestHbckMetricsResource.java

```diff
@@ -0,0 +1,260 @@
+      .header("X-Jersey-Tracing-Accept", true).get(String.class);
+    assertThat(response, allOf(containsString("\"hbck_report_start_time\":"),
+      containsString("\"hbck_report_end_time\":"), containsString("\"hbck_orphan_regions_on_fs\":"),
+      containsString("\"hbck_orphan_regions_on_rs\":"),
+      containsString("\"hbck_inconsistent_regions\":"), containsString("\"hbck_holes\":"),
+      containsString("\"hbck_overlaps\":"), containsString("\"hbck_unknown_servers\":"),
+      containsString("\"hbck_empty_region_info\":")));
+  }
+
+  @Test
+  public void testGetRootHtml() {
+    assertThrows(NotAcceptableException.class, () -> classRule.getTarget()
+      .request(MediaType.TEXT_HTML_TYPE).header("X-Jersey-Tracing-Accept", true).get(String.class));
+  }
+
+  @Test
+  public void testGetOrphanRegionOnFS() {
+    final String response =
+      classRule.getTarget().path("orphan_regions_on_fs").request(MediaType.APPLICATION_JSON_TYPE)
+        .header("X-Jersey-Tracing-Accept", true).get(String.class);
+    assertThat(response, allOf(startsWith("{\"data\":["), endsWith("]}")));
+  }
+
+  @Test
+  public void testGetOrphanRegionOnFSHtml() {
+    assertThrows(NotAcceptableException.class,
+      () -> classRule.getTarget().path("orphan_regions_on_fs").request(MediaType.TEXT_HTML_TYPE)
+        .header("X-Jersey-Tracing-Accept", true).get(String.class));
+  }
+
+  @Test
+  public void testGetOrphanRegionOnRS() {
+    final String response =
+      classRule.getTarget().path("orphan_regions_on_rs").request(MediaType.APPLICATION_JSON_TYPE)
+        .header("X-Jersey-Tracing-Accept", true).get(String.class);
+    assertThat(response, allOf(startsWith("{\"data\":["), endsWith("]}")));
+  }
+
+  @Test
+  public void testGetOrphanRegionOnRSHtml() {
+    assertThrows(NotAcceptableException.class,
+      () -> classRule.getTarget().path("orphan_regions_on_rs").request(MediaType.TEXT_HTML_TYPE)
+        .header("X-Jersey-Tracing-Accept", true).get(String.class));
+  }
+
+  @Test
+  public void testGetInconsistentRegions() {
+    final String response =
+      classRule.getTarget().path("inconsistent_regions").request(MediaType.APPLICATION_JSON_TYPE)
+        .header("X-Jersey-Tracing-Accept", true).get(String.class);
+    assertThat(response, allOf(startsWith("{\"data\":["), endsWith("]}")));
+  }
+
+  @Test
+  public void testGetInconsistentRegionsHtml() {
+    assertThrows(NotAcceptableException.class,
+      () -> classRule.getTarget().path("inconsistent_regions").request(MediaType.TEXT_HTML_TYPE)
+        .header("X-Jersey-Tracing-Accept", true).get(String.class));
+  }
+
+  @Test
+  public void testGetRegionHoles() {
+    final String response =
+      classRule.getTarget().path("region_holes").request(MediaType.APPLICATION_JSON_TYPE)
+        .header("X-Jersey-Tracing-Accept", true).get(String.class);
+    assertThat(response, allOf(startsWith("{\"data\":["), endsWith("]}")));
```

Comment:
instead of only checking for starts with "data". can we have some mocks to return a list of objects and validate if the response is proper? 
similarly at other places too.

Reply:
Done. Check https://github.com/apache/hbase/pull/5772/commits/8177556d139b5f7dddbb37ec55406d1172c2ede9